catalog/src/main/res/values/strings.xml

```diff
@@ -53,16 +53,18 @@
     <string name="component_checkbox_horizontal_group_title">Checkbox grouping with horizontal alignment</string>
     <string name="showkase_home_warning_title">⚠️ Theming don’t work on these!</string>
     <string name="showkase_home_warning_description">These are the previews that the developers have when working on components. They are not exhaustive and are only meant to give you a quick idea of what the component looks like.</string>
+    <!--Switch example-->
+    <string name="component_switch_description">Switch component allows the user to activate or deactivate the state of an element or concept. It is also used to control binary options (On/Off or True/False).</string>
 
     <string name="configurator_component_screen_title">Components configurations</string>
     <string name="configurator_component_screen_description">The Configurator aim to allow you to visualize a component in every possible state available state by customizing every property for each component.</string>
     <string name="configurator_component_screen_enabled_label">Enabled</string>
     <string name="configurator_component_screen_textfield_label">Label</string>
     <string name="configurator_component_screen_intent_label">Intent</string>
-    <!--Checkbox configurator-->
+    <!--Toggle configurator-->
```

Comment:
Toggle or Switch?

Reply:
I decided to rename it as this is relevant to all toggles not only switch or checkbox